.travis.yml

```diff
@@ -22,7 +22,7 @@ jdk:
 
 install: true
 
-script: travis_wait 60 ./gradlew test
+script: travis_wait 59 ./gradlew test
```

Comment:
Just changed this to have changes to commit. This commit would be reverted if Travis does not fail with  test timeout.